backend/main.py

```
@@ -185,46 +322,69 @@ async def event_generator():
-                "data": {"has_glaucoma": is_glaucoma, "confidence": confidence, "cup_to_disc_ratio": cup_to_disc_ratio, "image_base64": img_base64, "saved_to_db": bool(current_user)}
+                "data": {
+                    "has_glaucoma": is_glaucoma, 
+                    "confidence": confidence, 
+                    "cup_to_disc_ratio": cup_to_disc_ratio, 
+                    "image_base64": img_base64, 
+                    "image_url": image_url,
+                    "saved_to_db": bool(current_user)
+                }
             }) + "\n"
 
         except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)